plugins/MumblePlugin.h

```diff
@@ -1515,6 +1551,21 @@ struct MUMBLE_API_STRUCT_NAME {
 																			mumble_channelid_t channelID,
 																			const char **description);
 
+#if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+	/**
+     * Gets the positional audio data provided by OTHER plugins
+     *
+     * @param callerID The ID of the plugin calling this function
+     * @param[out] outPositionalData A pointer to the memory location the PositionalData object should be written to
+     * @returns The error code. If everything went well, STATUS_OK will be returned.
+     *
+     * @since Plugin interface v1.3.0
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *getPositionalAudioData)(mumble_plugin_id_t callerID,
+																		PositionalDataNoQt **outPositionalData);
```

Comment:
```suggestion
																		PositionalDataNoQt *outPositionalData);
```
No need to allocate the passed struct on the heap (using e.g. `new`). The only thing required is that the plugin allocates some space for this struct (by creating an instance on the Stack) and then the Mumble side can simply populate the fields of that struct with the correct data.

Reply:
Alright!